README.md

```diff
@@ -18,10 +18,11 @@ A Chart is a unit of Kubernetes manifests that reflect best practices as determi
 
 ## Install Helm
 
+To just use Helm, you can run the following:
```

Comment:
Right below this is "Using Helm" so I found this wording confusing. Maybe:
```
To install Helm, run this command:
```
nitpick! :green_book: